src/customize-pop/customize-desktop.md

```diff
@@ -0,0 +1,99 @@
+- [`Appearance` settings](/customize-pop/customize-desktop.md#appearance) provides the option to choose a dark or light theme that applies across the entire desktop environment.
+- [`Dock` settings](/customize-pop/customize-desktop.md#dock) provides option to modify Dock visibility, placement, size, and displayed icons.
+- [`Workspaces` settings](/customize-pop/customize-desktop.md#workspaces) includes options configure workspace positioning and behavior.
+
+---
+
+## Desktop Options
+
+Customize basic desktop elements in the `Desktop Options` tab.
+
+![General Settings](/images/customize-desktop/general-settings.png)
+
+| Setting | Function |
+|----------|----------|
+| Super Key Action | Assigns the `SUPER` key to start the Launcher, display Workspace navigation, or view installed applications. |
+| Hot Corner | Displays workspace navigation when the user moves the mouse cursor to the upper left corner of the screen. |
+| Top Bar | Display or hide items along the top bar of the main home screen. |
+| Window Controls | Enable or disable the minimize and maximize buttons in system and application windows. |
+
+## Background
+
+Choose a pre-installed desktop background or assign a custom image in the `Background` tab. You can also right-click anywhere on the desktop and select `Change Background...`.
+
+>**Note**: Background adjustments such as centered, scaled, spanned, stretched, and zoom can be applied using the GNOME Tweaks application: [GNOME Tweaks Appearance Settings](gnome-tweaks-extensions.md#appearance)
+
+![Background Tab](/images/customize-desktop/background-tab.png)
+
+### Set Pre-installed Images as Wallpaper
+
+Select an image from the list. There is no “apply” button; the desktop background will update automatically when an image is selected.
+
+![Set Pre-installed Image](/images/customize-desktop/set-preinstalled-image.png)
+
+### Set User Images as Wallpaper
+
+1. Click `Add Picture…` and navigate to the location of your image.
+
+    ![Click Add Picture](/images/customize-desktop/click-add-picture.png)
+
+2. Click `Open`.
+
+    ![Click Open](/images/customize-desktop/click-open.png)
+
+3. Select the image added to the list of available wallpapers.
+
+    ![Select Added Image](/images/customize-desktop/select-added-image.png)
+
+You can also right-click the image and choose `Set as Wallpaper` from within the file browser.
+
+![Set as Wallpaper](/images/customize-desktop/set-as-wallpaper.png)
+
+## Appearance
+
+Pop!\_OS includes two default themes: Light and Dark. These themes apply to all system and application windows across the operating system. Selecting a theme will also assign a default desktop background.
+
+**Light Theme**
+
+![Light Theme](/images/customize-desktop/light-theme.png)
+
+**Dark Theme**
+
+![Dark Theme](/images/customize-desktop/dark-theme.png)
+
+Enable the Light or Dark theme in the `Appearance` tab. Additional theme customizations can be applied using [GNOME Tweaks](gnome-tweaks-extensions.md).
```

Comment:
I don't think we necessarily want to encourage people to mess with their theme in GNOME Tweaks. This typically just leads to breakage. I liked that you separated the GNOME Tweaks call-out out into a side note earlier, so maybe consider doing that again here with something like this:

> **Note:** Custom themes may cause graphical glitches, but they can be applied using [GNOME Tweaks](gnome-tweaks-extensions.md).

Reply:
I'm glad you mentioned this. I know that Pop!_OS is intended to be a user-friendly, but modifiable if you have the technical know-how. That said, I'm not sure how in-depth "Customize Pop" should go. If we limit content to out-of-the-box settings, then maybe "Personalize Pop" is more accurate.

Reply:
I think "customize" vs. "personalize" is getting down to splitting hairs; "customize" is fine regardless of the scope we want to cover with it.

I'm not entirely opposed to covering GNOME Tweaks as long as it has the appropriate disclaimers. The current wording you applied from my comment looks good (aside from the typo of `cuase`), but I think the section as a whole is a little confusing since we have examples of the two built-in themes, the note about custom themes, then a picture of the built-in theme selector. I think the note about GNOME Tweaks should go at the end of the section, after the picture of the theme switcher. I also think the examples of the built-in themes need to look more subservient to the section as a whole.

I pushed 5ac5ffd3018ca1a45c8337380f4ccc9a70e87980 to try making it more clear what the examples are, feel free to undo or change it further (you can resolve this conversation if you're good with it.)